changelogs/api-management-service.md

```diff
@@ -1,6 +1,35 @@
 # Azure API Management service changelog
 
-Developer portal follows an independent release lifecycle and the [per-release changelog is available in the developer portal's GitHub repository](https://aka.ms/apimdevportal/releases).
+## Release - API Management service: May, 2022
+
+A regular Azure API Management service update was started on May 10, 2022. It may take several weeks for your API Management service to receive the update.
+
+### New features, fixes, and improvements
+
+1. Availability zone deployments are now available in the Switzerland North region.
+2. You can now access the API Management service name in runtime policies with the new `context.Deployment.ServiceId` property. The `ServiceId` property is now included in Application Insights logs.
+3. `validate-parameters` and `validate-content` policies now support GUID properties defined as `format: uuid`.
+
```

Comment:
5. (Public Preview) You can augment an existing GraphQL API using a REST API endpoint with the set-graphql-resolver policy.  You can also synthesize a GraphQL API by creating a GraphQL schema and providing aREST API endpoint for each resolver. 
6. Event Hub loggers can now use MSI authentication.
7. The json-to-xml policy allows the specification of namespaces and attributes in the JSON payload.